sympy/solvers/solvers.py

```diff
@@ -790,16 +816,16 @@ def solve(f, *symbols, **flags):
             A very fast, minimal testing.
         warn=True (default is False)
             Show a warning if ``checksol()`` could not conclude.
-        simplify=True (default)
+        simplify=True/False (default is None)
             Simplify all but polynomials of order 3 or greater before
             returning them and (if check is not False) use the
             general simplify function on the solutions and the
             expression obtained when they are substituted into the
             function which should be zero.
-        force=True (default is False)
+        force=True/False (default is None)
             Make positive all symbols without assumptions regarding sign.
```

Comment:
As far as I can tell `force` actually just defaults to `True`. We don't need a 3rd `None` value: we just need to note correctly that the default is `True` (and provide a better explanation of how `solve` uses this).